addons/calendar/tests/test_event_recurrence.py

```diff
@@ -352,6 +352,26 @@ def test_backward_pass_dst(self):
             (datetime(2023, 4, 27, 7, 00), datetime(2023, 4, 27, 8, 00)),
         ])
 
+    def test_all_day_date(self):
+        recurrence = self.env['calendar.event'].with_context(
+            default_start_date=date(2019, 10, 22),
+            default_stop_date=date(2019, 10, 22),
+        ).create({
+            'name': 'Recurrent Event',
+            'start': datetime(2019, 10, 22, 8, 0),
+            'stop': datetime(2019, 10, 22, 18, 0),
```

Comment:
When you check the context of the RPC of create, there are more keys. In order to have a relevant test, I would fullfill start_date,stop_date too